src/backend/app/Services/DatabaseProxyService.php

```diff
@@ -2,27 +2,28 @@
 
 namespace App\Services;
 
-use App\Models\DatabaseProxy;
+use App\Models\User;
 use App\Services\Interfaces\IBaseService;
 use Illuminate\Database\Eloquent\Collection;
```

Comment:
I'm surprised to see this still being here. I was expecting `DatabaseProxyService` to be removed as well

Reply:
I wasn't sure the best place to move some needed logic that lived inside `DatabaseProxyService`. 

From your comment below, I will move them to `CompanyService` and remove this entirely.